pyaedt/modeler/cad/Primitives.py

```diff
@@ -275,45 +587,4908 @@ def version(self):
+        if cs:
+            result = cs.create(
+                face=face,
+                origin=origin,
+                axis_position=axis_position,
+                axis=axis,
+                name=name,
+                offset=offset,
+                rotation=rotation,
+                always_move_to_end=always_move_to_end,
+            )
+
+            if result:
+                return cs
+        return False
+
+    @pyaedt_function_handler()
+    def create_object_coordinate_system(
+        self, obj, origin, x_axis, y_axis, move_to_end=True, reverse_x_axis=False, reverse_y_axis=False, name=None
+    ):
+        """Create an object coordinate system.
+
+        Parameters
+        ----------
+        obj : str, :class:`pyaedt.modeler.cad.object3d.Object3d`
+            Object to attach the object coordinate system to.
+        origin : int, VertexPrimitive, EdgePrimitive, FacePrimitive, list
+            Refer to the origin where the object coordinate system is anchored.
+            It can be:
+
+             - int in which case it refers to the entity Id.
+             - VertexPrimitive, EdgePrimitive, FacePrimitive in which case it refers to the entity type.
+             - list in which case it refers to the origin coordinate system ``[x, y, z]``.
+
+        x_axis : int, VertexPrimitive, EdgePrimitive, FacePrimitive, list
+            Entity that the x axis of the object coordinate system points to.
+            It can be:
+
+             - int in which case it refers to the entity Id.
+             - VertexPrimitive, EdgePrimitive, FacePrimitive in which case it refers to the entity type.
+             - list in which case it refers to the point coordinate system ``[x, y, z]`` that the x axis points to.
+
+        y_axis : int, VertexPrimitive, EdgePrimitive, FacePrimitive, list
+            Entity that the y axis of the object coordinate system points to.
+            It can be:
+
+             - int in which case it refers to the entity Id.
+             - VertexPrimitive, EdgePrimitive, FacePrimitive in which case it refers to the entity type.
+             - list in which case it refers to the point coordinate system ``[x, y, z]`` that the y axis points to.
+
+        move_to_end : bool, optional
+            If ``True`` the Coordinate System creation operation will always be moved to the end of subsequent
+            objects operation. This will guarantee that the coordinate system will remain solidal with the object
+            face. If ``False`` the option "Always Move CS to End" is set to off. The default is ``True``.
+        reverse_x_axis : bool, optional
+            Whether the x-axis is in the reverse direction.
+            The default is ``False``.
+        reverse_y_axis : bool, optional
+            Whether the y-axis is in the reverse direction.
+            The default is ``False``.
+        name : str, optional
+            Name of the coordinate system. The default is ``None``.
```

Comment:
What is the behavior that occurs when ``None`` is used. Extend the sentence like this: "The default is ``None``, in which case a name is automatically assigned."